frontends/web/src/new_front/components/Tables/Leaderboard/PairLanguageLeaderboard.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+import { Dropdown, DropdownButton } from "react-bootstrap";
+import BootstrapTable from "react-bootstrap-table-next";
+import paginationFactory from "react-bootstrap-table2-paginator";
+
+const PairLanguageLeaderboard = () => {
+  const [selectedLanguage, setSelectedLanguage] = useState("en-fr");
+
+  const pairLanguages = [
+    "en-fr",
+    "en-de",
+    "en-es",
+    "en-ru",
+    "en-zh",
+    "en-tr",
+    "en-ar",
+    "en-ja",
+    "en-ko",
+    "en-pt",
+    "en-it",
+  ];
+
+  const resultModels = [
```

Comment:
What are this constants for?

Reply:
This is mock data, in a near nearr future (this weekend) will be solved

Reply:
Sounds good